web/source/kmwuitoolbar.ts

```diff
@@ -253,6 +253,7 @@ if(!window['keyman']['ui']['name']) {
       aNode = ui.createNode('a', null, 'kmw_button_a');
       aNode.href = '#';
       aNode.onclick = ui.showOSK;
+      aNode.onmousedown = function() { keymanweb['activatingUI'](true); };
```

Comment:
```suggestion
      aNode.onmousedown = function() {
        keymanweb['activatingUI'](true);
      };
```

I'd like to try and not perpetuate the bad old coding styles :grin: